### DISCUSSION TITLE
为什么谷歌商城版插件和导入安装会有功能上的差异

### DISCUSSION THREAD

**Discussion by szh-bash** [nextai-translator/nextai-translator#276] [Q&A] [answered]
例如 #170 #241 @yetone

**Comment by yetone** [comment, marked as answer]
因为 Chrome Web Store 的审核太慢了，一直落后我若干个版本

**Reply by abc4329803** [reply]
那请问我怎么才能在chrome直接安装最新版本呢？

**Reply by szh-bash** [reply]
> 那请问我怎么才能在chrome直接安装最新版本呢？

release 页面下载最新版 extension.zip，解压后导入至 chrome，具体可以google或者baidu一下
